app/assets/stylesheets/components/page_specific/_content_hub.scss

```diff
@@ -36,15 +36,14 @@
 .content-hub__hero-image {
   padding-bottom: 100%;
```

Comment:
I would suggest making this percentage value equal to the aspect ration of the image so that it fits the space exactly. In this case that can be `250 / 479 * 100%`. This helps with the positioning of the elements that follow in the flow.